massgov/mayflower
# Restructured to include and link to core

## Description
Re-structure the gatsby site so that it is a single point of entry for Mayflower that removes the link to gitbook, makes core central, and links to docs for the other packages but labels them as experimental.

-Simplified layout
-Labelled react and patternlab as experimental
-Changed the main content to mirror core docs and deep-linked to them
-Simplified styling to be more compact
-Labelled examples with the package they use
-simplified headline to combine commonwealth and system
-Re-worded description to emphasize to Mass brand and added apps

## Related Issue / Ticket
- [JIRA issue](DP-20573)

## Steps to Test
1. Check out the branch and  run rush start:site
2. Open a browser to localhost:8000
3. Compare to screenshots

## Screenshots
<img width="1341" alt="Screen Shot 2021-01-08 at 2 44 37 PM" src="https://user-images.githubusercontent.com/3894652/104062747-9d538800-51c0-11eb-9a54-8c5ec45b258f.png">
<img width="1378" alt="Screen Shot 2021-01-08 at 2 44 49 PM" src="https://user-images.githubusercontent.com/3894652/104062756-a0e70f00-51c0-11eb-90ed-fbfb30961b47.png">
<img width="1304" alt="Screen Shot 2021-01-07 at 12 19 03 PM" src="https://user-images.githubusercontent.com/3894652/103929319-c0a80580-50e2-11eb-8e56-0d54e4a29939.png">
